packages/dx-chart-core/src/plugins/series/computeds.ts

```diff
@@ -141,15 +166,23 @@ const getRect = (cx: number, cy: number, dx: number, dy: number): Rect => (
 );
 
 getBarPointTransformer.getTargetElement = (point) => {
-  const { x, y, y1, barWidth, maxBarWidth } = point as BarSeries.PointProps;
-  const width = barWidth * maxBarWidth;
-  const height = Math.abs(y1! - y);
-  return getRect(x, y + height / 2, width / 2, height / 2);
+  const {
+    arg, val, startVal, barWidth, maxBarWidth, rotated,
+  } = point as BarSeries.PointProps;
+  const halfWidth = barWidth * maxBarWidth / 2;
+  const halfHeight = Math.abs(startVal! - val) / 2;
+  const centerVal = (val + startVal!) / 2;
+  return getRect(
```

Comment:
I think we should add `rotated` as the last argument of `getRect` and perform flips there. It'll make code clearer